pgx/bridge_bidding.py

```diff
@@ -97,8 +98,11 @@ class BridgeBidding(v1.Env):
         # fmt: on
+        self.dds_hash_table: str = os.getcwd() + "/dds_hash_table"
```

Comment:
```suggestion
        dds_hash_table: str = os.getcwd() + "/dds_hash_table"
```

- `self` にする意味がない。 
- `os.path.join` を使う（以下も同様）